src/Logging/composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "google/cloud-logging",
+    "description": "Stackdriver Logging Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "require": {
+        "google/cloud-core": "*"
+    },
```

Comment:
Those should bubble up from the required dependency on [cloud-core](https://github.com/GoogleCloudPlatform/google-cloud-php/pull/360/files#diff-54380dbdca67ea57cc29362d78985a7cR15).